roles/app/defaults/main.yml

```diff
@@ -3,7 +3,7 @@ s3_headers: "{\"Cache-Control\":\"max-age=31557600\"}"
 s3_protocol: "https"
 
 attachment_url: "/spree/products/:id/:style/:basename.:extension"
-attachment_styles: "{\"mini\":[\"48x48>\",\"\"],\"small\":[\"100x100>\",\"\"],\"product\":[\"240x240>\",\"\"],\"large\":[\"600x600>\",\"\"]}"
+attachment_styles: "{\"mini\":[\"48x48#\",\"\"],\"small\":[\"227x227#\",\"\"],\"product\":[\"240x240>\",\"\"],\"large\":[\"600x600>\",\"\"]}"
```

Comment:
Great!

Can we add a comment to describe the usage of these styles?

- mini: used in the backend, bulk product edit and order cycle edit
- small: used in the shopfront
- product: product image upload modal and edit product image page
- large: product modal

I agree that `mini` should be cropped as well. It needs to fit in the table design and shows admins straight away how it will look like in the shop front. 

It's out of scope of this PR but the `product` style doesn't make much sense. It would be great to improve the admin interface with @Erioldoesdesign's help. But I guess that has to go to a wishlist item first.

Reply:
Shall we convert everything to JPG in this PR as well? I think it would result in much smaller files and regenerating all images takes a whole day (last time I ran it for Australia). I think we have around 10GB in product image data and Australia uses JPG already.

Reply:
yeah, I added the comments :+1:

I also changed to convert everything to jpg on all servers.